docs/guides/model_surgery.ipynb

```diff
@@ -0,0 +1,412 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    flat_params = traverse_util.flatten_dict(params)
</pre><p>I'd consider using the " sep='/' " keyword argument to make the flattened keys simple strings as opposed to tuples, which is usually nicer for most users</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/2668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I'd consider doing this across the entire doc using the concatenated string "foo/bar/baz" flattened form.

Reply:
That's a good idea. Though it worth mentioning that people can use tuples as key - which is slightly more error-proof because it allows '/' as part of the dict keys. I will add it in this doc.